sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/fl_scatter_gather.py

```diff
@@ -44,8 +43,31 @@
 class FLScatterGather(ControlFlowNode, NodeIOMixin):
     """A node which creates a federated learning scatter-gather loop as a pipeline subgraph.
     Intended for use inside a pipeline job. This is initialized when calling
-    dsl.fl_scatter_gather() or when loading a serialized version of this node from YAML.
+    `dsl.fl_scatter_gather()` or when loading a serialized version of this node from YAML.
     Please do not manually initialize this class.
+
+    :param silo_configs: List of federated learning silo configurations.
+    :type silo_configs: List[~azure.ai.ml.entities._assets.federated_learning_silo.FederatedLearningSilo]
+    :param silo_component: Component representing the silo for federated learning.
+    :type silo_component: ~azure.ai.ml.entities._component.component.Component
+    :param aggregation_component: Component representing the aggregation step.
+    :type aggregation_component: ~azure.ai.ml.entities._component.component.Component
+    :param aggregation_compute: The compute resource for the aggregation step.
+    :type aggregation_compute: str, optional
```

Comment:
I made the modifications strictly according to the Sphinx style, and if I follow what you said, I will need to do a lot of work.
I think if the `Optional [str]` format is necessary, it can be clearly stated in the refer doc.

https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#the-sphinx-docstring-format.
![image](https://github.com/Azure/azure-sdk-for-python/assets/23182548/0495531a-2558-45f4-b81d-19d5be5f0168)

Reply:
Okay, we can leave it this way. It won't be consistent throughout the codebase, but your point is fair that it should be intelligible by any user. Thanks for informing me about the format.

Reply:
I did some more reading. The distinction is that "<type>, optional" is the format for when the parameter itself isn't required for the function to be called. However, "Optional[<type>]" indicates that the value can be None. Just FYI.